webapi/src/test/scala/org/knora/webapi/responders/v2/ValuesResponderV2Spec.scala

```diff
@@ -220,56 +219,66 @@ class ValuesResponderV2Spec extends CoreSpec() with ImplicitSender {
   private def checkValueIsDeleted(
     resourceIri: IRI,
     maybePreviousLastModDate: Option[Instant],
-    propertyIriForGravsearch: SmartIri,
-    propertyIriInResult: SmartIri,
     valueIri: IRI,
     customDeleteDate: Option[Instant] = None,
```

Comment:
I would call it `deleteDate` instead of `customDeleteDate` (I see that it wasn't you who named it like that).